docs/authoring-tasks.adoc

```diff
@@ -113,13 +117,13 @@ pipeline:
       workspace: shared-workspace
 ----
 
-Notice that the `taskRef.kind` is set to `Task` instead of `ClusterTask` (which is used for tasks which are provided by `ods-pipeline`).
+Notice that the `taskRef.kind` is set to `Task`, not `ClusterTask`.
```

Comment:
I think we can remove this sentence completely - the default is `Task` even for the tasks that come with the installation ...